src/middlewared/middlewared/api/base/server/doc.py

```diff
@@ -20,13 +22,20 @@ class APIDumpMethod(BaseModel):
     schemas: dict
 
 
+class APIDumpEvent(BaseModel):
+    name: str
```

Comment:
Can we add doc strings for each field so it shows up in the docs?

Reply:
This is for internal use, it does not show up in the docs